Makefile

```diff
@@ -137,7 +131,7 @@ ci-data-dependencies:
 		--run 'export LANG=C.UTF-8; packell data-dependencies validate'
 
 .PHONY: ci-local
-ci-local: clean-all ci-headers-check ci-versioning ci-data-dependencies ci-build ci-validate ci-build-java ci-build-js ci-test ci-docs
```

Comment:
Currently the `ci-docs` target does a few things:
- run `daml damlc docs` to generated the code documentation
- build the theme
- run sphinx to convert `rst` to `html`

We would still need the first step to produce the set of `rst` that then need to be packaged as part of the assembly build

Reply:
Sorry I missed the dependency on `doc-code` (which is still there) in `doc-html`. I'm not super familiar with Makefiles.

Does b7a8bdd29afa45affedf3395391948ef77e8acc4 address this appropriately?